### DISCUSSION TITLE
扩展 Shortcodes 总览 | FixIt

### DISCUSSION THREAD

**Discussion by giscus** [hugo-fixit/FixIt#199] [General]
# 扩展 Shortcodes 总览 | FixIt

FixIt 主题在 Hugo 内置的 shortcode 的基础上提供多个扩展的 shortcode

https://fixit.lruihao.cn/zh-cn/docs/content-management/shortcodes/extended/introduction/

**Comment by hzyangjc** [comment]
image类型的height和width的语法是怎么样的？
尝试了几种都还是塞满整个页面XD

**Reply by Lruihao** [reply]
> 这是一个 [bug#200](https://github.com/hugo-fixit/FixIt/issues/200)

已修复，[范例](https://pre.fixit.lruihao.cn/tests/shortcodes/#image)

**Comment by zyyaria** [comment]
请问怎么创建这种 [Tab 标签](https://www.docsy.dev/docs/adding-content/shortcodes/#tabbed-panes)？
[![x8ke1J.png](https://s1.ax1x.com/2022/10/07/x8ke1J.png)](https://imgse.com/i/x8ke1J)
我从 [Hugo-Docsy-layouts-shortcodes](https://github.com/google/docsy/tree/main/layouts/shortcodes) 复制 scss 和 html 过来，但是显示会变成下面这样子：
[![x8kmc9.png](https://s1.ax1x.com/2022/10/07/x8kmc9.png)](https://imgse.com/i/x8kmc9)
不知道咋整的

**Reply by Lruihao** [reply]
[#634](https://github.com/hugo-fixit/FixIt/pull/634) 已新增 tabs 和 tab 支持。
